daemon/inertiad/notifier/notifier.go

```diff
@@ -0,0 +1,55 @@
+package notifier
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+)
+
+// Notifier manages notifications
+type Notifier interface {
+	Notify(string) error
+}
+
+// SlackNotifier represents slack notifications
+type SlackNotifier struct {
+	hookURL string
+}
+
+// NewNotifier creates a notifier with web hook url to slack channel
+func NewNotifier() *SlackNotifier {
+	url := "https://hooks.slack.com/services/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"
+
+	n := &SlackNotifier{
+		hookURL: url,
+	}
+
+	return n
+}
+
+// Notify sends the notification
+func (n *SlackNotifier) Notify(text string) error {
+	message := map[string]interface{}{
+		"text": text,
+	}
+
+	bytesRepresentation, err := json.Marshal(message)
+	if err != nil {
+		return err
```

Comment:
It's good practice to include additional context in your errors:

```suggestion
		return fmt.Errorf("failed to encode request: %s", err.Error())
```